modules/integration-tests/src/main/java/org/apache/fluo/integration/impl/WorkerIT.java

```diff
@@ -51,6 +51,10 @@
 
   private static Column observedColumn = LAST_UPDATE;
 
+  private static void setObservedColumn(Column c) {
```

Comment:
You might be able to avoid creating this method if you just make the WorkerITObserverProvider inner class (and the observedColumn member) non-static.

Reply:
I tried this and it did not work because Fluo needs to be able to instantiate that class.

Reply:
Okay, got it. Guess this works, then.